src/api/migrations/20190708141337-create-stat.js

```diff
@@ -0,0 +1,33 @@
+export default {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Reads', {
```

Comment:
@alainburindi what do you think about a stats table that just holds the details of the article and then all stats such as readCount, shareCount, commentCount etc for each article is stored in that one table.

These stats would then get incremented every time an article is viewed, shared or commented on.